plutus-contract/src/Language/Plutus/Contract/Effects/WriteTx.hs

```diff
@@ -83,8 +83,8 @@ writeTxConfirmed
   , Req.AsContractError e
   )
   => UnbalancedTx
-  -> Contract s e TxId
-writeTxConfirmed t = writeTxSuccess t >>= awaitTxConfirmed
+  -> Contract s e ()
+writeTxConfirmed t = void (writeTxSuccess t >>= awaitTxConfirmed)
```

Comment:
This is backwards from what I meant! I meant that `awaitTxConfirmed` should return `()`, since it doesn't give you any new information, but that `writeTxConfirmed` should return the id for convenience.

Reply:
Aaah I see!